doc/funcref.sgml

```diff
@@ -3450,6 +3458,72 @@ returns one of the constants<itemize>
 </quote>
 
 
+<sect1>getcurrentdevice<label id="getcurrentdevice"><p>
+
+<quote>
+<descrip>
+<tag/Function/Get current device.
+<tag/Header/<tt/<ref id="device.h" name="device.h">/
+<tag/Declaration/<tt/unsigned char getcurrentdevice (void);/
+<tag/Description/The function returns the current device.
```

Comment:
Please add "It allows to access the current device with the 'Low-level disk I/O API' or cbm_* I/O functions requiring a 'device' parameter."